### PR TITLE
[12.0][MIG] stock_quantity_history_location

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

CC @Eficent

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

File: stock_quantity_history_location/wizards/stock_quantity_history.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        action = super(StockQuantityHistory, self).open_table()
+        ctx = action['context']
+        if isinstance(ctx, str):
+            ctx = ast.literal_eval(ctx)
+        action['domain'] = [('type', '=', 'product')]
+        if self.location_id:
+            ctx['location'] = self.location_id.id
+            ctx['compute_child'] = self.include_child_locations
+            if ctx.get('company_owned', False):
+                ctx.pop('company_owned')
```
Comment (rousseldenis) [inline stock_quantity_history_location/wizards/stock_quantity_history.py:28]: @hveficent It seems that tests do not get here. Could you see  why ?

Comment (HviorForgeFlow) [reply stock_quantity_history_location/wizards/stock_quantity_history.py:28]: Test now cover that case.

**Comment by HviorForgeFlow** [comment]
@rousseldenis can you update your review?

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request closed without merging**